frontends/election-manager/src/screens/election_smartcards_screen.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { LinkButton, Prose } from '@votingworks/ui';
+
+import { CardProgrammingPrompt } from '../components/card_programming_prompt';
+import { NavigationScreen } from '../components/navigation_screen';
+import { routerPaths } from '../router_paths';
+
+export function ElectionSmartcardsScreen(): JSX.Element {
+  return (
+    <NavigationScreen>
+      <Prose maxWidth={false} textCenter>
```

Comment:
I like the new prompt with the image. Definitely more clear.

I do think it's a bit weird to have a centered layout in a tab when all the other tabs have left aligned content. Not totally sure what to do about it.

Reply:
Yeah I went back and forth on this as well and tried a few permutations, e.g. left aligning the "Smartcards" header while centering everything else, centering only the graphic, left aligning everything, etc.

Will do a little more timeboxed exploration and will then wait for guidance from @beausmith, merging this and making tweaks later if necessary

Reply:
Given discussion at the prod/eng meeting, let me post a question with some options in #vxsuite-prodeng to get some quick feedback from Roe!

Reply:
Switched to left alignment for consistency with other tabs and got confirmation from Roe on the decision! Images in PR description updated accordingly